### DISCUSSION TITLE
Welcome to mxgraph Discussions!

### DISCUSSION THREAD

**Discussion by coclav** [maxGraph/maxGraph#11] [Announces]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by junsikshim** [comment]
Hello, my name is Junsik, and I've been enjoying the experience of using mxGraph.
I'd like to contribute where I can, and discuss future possibilities of this library.
Thanks!

**Comment by CommunityMan** [comment]
Hi everyone,
I'm Quentin, I am mainly a java developer, linux fanboy. Working more in JavaScript in the last two years, migrating our UI at work to webcomponent.
I modified mxGraph in that context to use it more as a flow editor with metadata. Instead of using it only for visual properties.
I'm really interested to pass it to es6 as it would gives it module capabilities as well as webcomponent capabilities which could help tweaking the UI more easily (mainly the graph editor)
Nice to meet you all,
Thank for your contribution to this great project
